docs/connectors/configuration-parameters/record-format.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Output Formats"
-sidebar_position: 5
+title: "Record format"
```

Comment:
IMO we should still have "output" somewhere in the title, it makes it easier to find if you're trying to figure out how to write out the record in a specific format.

Reply:
@lovromazgon works for me